pkg/kdk/snapshot.go

```diff
@@ -21,14 +21,11 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/client"
 )
 
-func Snapshot(ctx context.Context, dockerClient *client.Client, logger logrus.Entry) error {
-
-	snapshotName := KdkConfig.AppConfig.Name + "-" + strconv.Itoa(int(time.Now().UnixNano()))
-
-	_, err := dockerClient.ContainerCommit(context.Background(), KdkConfig.AppConfig.Name, types.ContainerCommitOptions{Reference: snapshotName})
+func Snapshot(cfg KdkEnvConfig, logger logrus.Entry) error {
+	snapshotName := cfg.ConfigFile.AppConfig.Name + "-" + strconv.Itoa(int(time.Now().UnixNano()))
```

Comment:
This is kind of hard to read. 

May: break it up using variables.

Reply:
Done.